mozilla/foundation.mozilla.org
# updated author-images

Closes #7314 


**Link to sample test page**: https://foundation-s-7314-autho-ns6qxz.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

**Steps to test:**

1. Visit the page linked above
2. Double check that the author images are displaying correctly
3. If you would like to make any updates to the authors, you can log into the CMS using the following credentials (admin1/admin1)
4. If everything is appearing correctly, testing is complete! 


## Checklist

**Changes in Models:**
